source/MRMesh/MRZlib.cpp

```diff
@@ -0,0 +1,131 @@
+#include "MRZlib.h"
+#include "MRDeferred.h"
+
+#include <zlib.h>
+
+namespace
+{
+
+constexpr size_t cChunkSize = 256 * 1024; // 256 KiB
+
+std::string zlibToString( int code )
+{
+    switch ( code )
+    {
+        case Z_OK:
+            return "OK";
```

Comment:
why in upper case and all other in lower case?

Reply:
Fixed.